src/main/startup/system_stm32g4xx.c

```diff
@@ -450,4 +450,7 @@ void SystemClock_Config(void)
   pInit.HSI48CalibrationValue = 32;
 
   HAL_RCCEx_CRSConfig(&pInit);
+
+  /* Configure Flash prefetch, Instruction cache, Data cache and wait state */
+  FLASH->ACR = FLASH_ACR_DBG_SWEN | FLASH_ACR_PRFTEN | FLASH_ACR_ICEN |FLASH_ACR_DCEN |FLASH_ACR_LATENCY_4WS;
```

Comment:
```suggestion
  FLASH->ACR = FLASH_ACR_DBG_SWEN | FLASH_ACR_PRFTEN | FLASH_ACR_ICEN | FLASH_ACR_DCEN | FLASH_ACR_LATENCY_4WS;
```

Reply:
This was copied from the F4 code, lack of spaces included.